pkg/reconciler/instances/connectivityproxy/command.go

```diff
@@ -0,0 +1,86 @@
+package connectivityproxy
+
+import (
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/chart"
+	"github.com/kyma-incubator/reconciler/pkg/reconciler/service"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	apiCoreV1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+//go:generate mockery --name=Commands --output=mocks --outpkg=connectivityproxymocks --case=underscore
+type Commands interface {
+	Install(*service.ActionContext, *apiCoreV1.Secret) error
+	CopyResources(context *service.ActionContext) error
+	Remove(context *service.ActionContext) error
+}
+
+type NewInClusterClientSet func(logger *zap.SugaredLogger) (kubernetes.Interface, error)
+type NewTargetClientSet func(context *service.ActionContext) (kubernetes.Interface, error)
+
+type CommandActions struct {
+	clientSetFactory       NewInClusterClientSet
+	targetClientSetFactory NewTargetClientSet
+	install                service.Operation
+	copyFactory            []CopyFactory
+}
+
+func (a *CommandActions) Install(context *service.ActionContext, bindingSecret *apiCoreV1.Secret) error {
+	for key, value := range bindingSecret.Data {
+		configuration := context.Model.Configuration
+		if configuration == nil {
+			context.Model.Configuration = make(map[string]interface{})
```

Comment:
Please add this check and the creation of an empty map to the [invoker.go](https://github.com/kyma-incubator/reconciler/blob/main/pkg/scheduler/invoker/invoker.go#L25) - we should not delegate such checks to our comp-reconcilers